### PR TITLE
Capitalized

### LINKED ISSUE #157: replace usage of StringsInCamelCase with `Capitalized` from `cactoos.org`

### PR DESCRIPTION
**Pull request by l3r8yJ**
closes #157 

@h1alexbel take a look, please

### CODE REVIEW

File: src/main/java/io/github/eocqrs/kafka/xml/NameInCamelCase.java [hunk 1]
```diff
@@ -23,10 +23,15 @@
...
+import lombok.SneakyThrows;
+import org.cactoos.Text;
+import org.cactoos.func.FuncOf;
+import org.cactoos.iterable.IterableOf;
 import org.cactoos.list.ListOf;
-import org.cactoos.text.Concatenated;
-import org.cactoos.text.Joined;
-import org.cactoos.text.TextOf;
+import org.cactoos.proc.ForEach;
+import org.cactoos.text.*;
```
Comment (h1alexbel) [inline src/main/java/io/github/eocqrs/kafka/xml/NameInCamelCase.java:32]: don't use *, please

File: src/main/java/io/github/eocqrs/kafka/xml/NameInCamelCase.java [hunk 2]
```diff
@@ -23,10 +23,15 @@
...
+
+import java.util.stream.Collectors;
 /**
```
Comment (h1alexbel) [inline src/main/java/io/github/eocqrs/kafka/xml/NameInCamelCase.java:35]: Remove puzzle

File: src/main/java/io/github/eocqrs/kafka/xml/NameInCamelCase.java (hunk 1 above)
Comment (l3r8yJ) [reply src/main/java/io/github/eocqrs/kafka/xml/NameInCamelCase.java:32]: @h1alexbel I agree, my IDEA is annoyed by this import section

**Comment by l3r8yJ** [comment]
@h1alexbel take a look, please

i'm not able to resolve
image

**Review by h1alexbel — APPROVED** [review]

**Comment by l3r8yJ** [comment]
@h1alexbel seems like nothing has changed, but for some reason now everything is fine and we can merge, please check

**Comment by h1alexbel** [comment]
@rultor merge

**Comment by h1alexbel** [comment]
@l3r8yJ thanks!!

**Pull request merged**